lms/envs/test.py

```diff
@@ -261,6 +261,14 @@
 FAKE_EMAIL_DOMAIN = 'fake-email-domain.foo'
 THIRD_PARTY_AUTH_FALLBACK_FULL_NAME = "Unknown"
 
+THIRD_PARTY_AUTH_CUSTOM_AUTH_FORMS = {
+    'custom1': {
+        'secret_key': 'opensesame',
```

Comment:
@bradenmacdonald side note: opensesame sounds like a name of another open-source javascript testing framework :)